libraries/tools/kotlin-stdlib-gen/src/templates/Elements.kt

```diff
@@ -811,6 +811,10 @@ object Elements : TemplateGroupBase() {
         includeDefault()
         include(Lists, CharSequences, ArraysOfUnsigned)
     } builder {
+        specialFor(CharSequences) {
+            inlineOnly()
```

Comment:
Is there a reason to use it here?

Reply:
I could not understand how this works... I will delete this.

Reply:
Delete this: https://github.com/JetBrains/kotlin/pull/4871/commits/bce231a8be4dc97e07424b3635b708a11cf25c85